includes/admin-reports/add_new_attendee.php

```diff
@@ -33,11 +33,11 @@ function add_new_attendee($event_id){
         wp_print_scripts('reCopy');
 
 	global $wpdb;
-	$sql  = "SELECT * FROM " .EVENTS_DETAIL_TABLE. " WHERE is_active='Y' AND event_status != 'D' AND id = '" . $event_id . "' LIMIT 0,1";
-
+	$sql  = "SELECT * FROM " .EVENTS_DETAIL_TABLE. " ed WHERE ed.is_active='Y' AND ed.event_status != 'D' AND ed.id = '%d' LIMIT 0,1";
+	
 	//Build the registration page
-	if ($wpdb->get_results($sql)){
-			$events = $wpdb->get_results($sql);
+	if ($wpdb->get_results($wpdb->prepare( $sql, $event_id ), OBJECT_K )){
+			$events = $wpdb->get_results($wpdb->prepare( $sql, $event_id ), OBJECT_K );
```

Comment:
This does the same query twice, which is inefficient and prone to errors when refactoring.
I'd suggest instead something like
```
$events = $wpdb->get_results($wpdb->prepare( $sql, $event_id ), OBJECT_K );
if($events){...
```